mcts.py

```diff
@@ -207,7 +209,24 @@ def incorporate_results(self, move_probabilities, value, up_to):
             self.revert_visits(up_to=up_to)
             return
         self.is_expanded = True
-        self.original_prior = self.child_prior = move_probabilities
+
+        '''
+        temp = move_probabilities * (1 - self.illegal_moves)
```

Comment:
is this ready for review?

Reply:
this was here to demonstrate my methodology for determining rough impact I should have commented "WILL BE REMOVED BEFORE SUBMIT"